index.md

```diff
@@ -269,6 +268,8 @@
 
 [Terms of Service; Didn't read](https://tosdr.org/) :c-extension: :c-resource: - Resource to get a quick summary on terms of service, as well as an overall privacy and user freedom score
 
+[Tailscale](https://tailscale.com/) :c-pc: :c-mobile: :c-freemium: - Zero-config WireGuard VPN to remotely access a device's services anywhere, with a dedicated web SSH client for devices advertising as Tailscale SSH available, can also advertise as an 'Exit Node' on some devices to route internet traffic through said Exit Node.
```

Comment:
1.  Could we remove the part "advertising as Tailscale SSH available, can also advertise as an 'Exit Node' on some devices to route internet traffic through said Exit Node."?

2. Tailscale can be selfhostable? If yes, then add the proper icon

Reply:
1. why?
2. nope, at least if it can only for enterprises

Reply:
1. too technical. simplicity is good, after all the one that will read doesn't know anything about tailscale, hell what is a "exit node"? (don't respond, it is to illustrate)
2. so its a wireguard provider basically?

Reply:
yeah it is just a wireguard provider basically, you can just change it as you see fit but its basically just a way to remote access your devices over an advanced wireguard vpn called a 'tailnet', highly customisable

Reply:
well, i'll do changes over it. seems to under the providers section

Reply:
@enhancedrock check the changes. anything that i missed?

Reply:
might be worth adding "designed with the purpose of easily remotely accessing (devices/servers/infrastructure - up to you)"?

Reply:
is it hard to access with wireguard? i thought tailscale was easy to configure, not easy to "access the stuff"

Reply:
i decided to remove tailscale. it is just a wireguard provider with features to make it easier due to its centralization, which for me the centralization alone makes it ring the red alarm on my head. i open an exception for proton vpn and mail because their business model relies on the consumer's trust, and their features shows that

also zerotier is selfhostable mostly (you cant host their UI from what i heard, but everything else you can), so we may look towards this instead